tests/unit/Term/AliasGroupListTest.php

```diff
@@ -191,12 +184,10 @@ public function testGivenEmptyGroup_setGroupRemovesGroup() {
 			new AliasGroup( 'en', array( 'foo' ) ),
 		) );
 
-		$expectedList = new AliasGroupList();
-
 		$list->setGroup( new AliasGroup( 'en' ) );
 		$list->setGroup( new AliasGroup( 'de' ) );
 
-		$this->assertEquals( $expectedList, $list );
+		$this->assertTrue( $list->isEmpty() );
```

Comment:
I'm sorry? So you withdraw your original argument to avoid "binding to more production code"? Because `assertEquals` binds to **all** production code.